easse/cli.py

```diff
@@ -152,16 +157,50 @@ def report(
         sys_sents_path=None,
         orig_sents_path=None,
         refs_sents_paths=None,
-        report_path='report.html',
+        report_path='easse_report.html',
         tokenizer='13a',
         metrics=','.join(DEFAULT_METRICS)
         ):
     """
     Create a HTML report file with automatic metrics, plots and samples.
     """
-    orig_sents, sys_sents, refs_sents = get_sents(test_set, orig_sents_path, sys_sents_path, refs_sents_paths)
+    sys_sents = get_sys_sents(test_set, sys_sents_path)
+    orig_sents, refs_sents = get_orig_and_refs_sents(test_set, orig_sents_path, refs_sents_paths)
     lowercase = is_test_set_lowercase(test_set)
     write_html_report(
-            report_path, orig_sents, sys_sents, refs_sents, test_set_name=test_set,
+            report_path, orig_sents, sys_sents, refs_sents, test_set=test_set,
+            lowercase=lowercase, tokenizer=tokenizer, metrics=metrics,
+            )
+
+
+@cli.command('multiple_systems_report')
```

Comment:
Is it necessary to have another different command? Couldn't we pass an argument (like `--multiple-systems`, for example) to the `report` command?

Reply:
Good point. We could even infer that if multiple paths are sent to sys_sents_path, what do you think ?

Reply:
Ok I took care of it